analyses/survival-analysis/survival-analysis_subtypes.Rmd

```diff
@@ -167,11 +182,43 @@ Save the plot to a file.
 
 ```{r Save survival plot}
 # We can save the plot like a normal ggplot
-cowplot::save_plot(filename = kap_meier_plot_file, plot = surv_plot_all_subtype)
+cowplot::save_plot(filename = kap_meier_plot_file, plot = surv_plot_all_subtype, base_height = 4,  base_width = 8)
 # Let's save the model for CI and median survial as well
 readr::write_tsv(surv_median(kap_fit$model), kap_meier_model_file)
 ```
 
+## Analysis: cox regression for HGG molecular subtypes
+We observe a significant poor prognostic effect of TP53 loss in tumors with H3 K28 mutations.
+
+```{r cox hgg subtype}
+# Check Ns
+as.data.frame(table(metadata_ind_dna$OS_status, metadata_ind_dna$molecular_subtype))
+
+# Run model
+fit_save_model(metadata_ind_dna, 
+               "molecular_subtype",
+               file.path(results_dir, "cox_hgg_subtype.RDS"))
+```
+
+Why are we not seeing H3 K28 subtypes?
```

Comment:
Since this subtype is in the model, this should be rephrased or chunk updated accordingly.